### PR TITLE
Fixed tooltip CSS classes

### LINKED ISSUE #600: GOCO graph tooltips styling messed up
Relatively minor, essentially just really bad kerning (no padding between tootlip items, haha).

### PR DESCRIPTION
**Pull request by AmndaPlant**
Closes #600

GOCO and a few other files seemed to be using an old CSS class name (`nivo-tooltip__content` which was changed in commit ff55eaf which was causing their spacing to be a bit messed up, updating the class names fixed this.

### CODE REVIEW

File: client/src/charts/wrapped_nivo/nivo_bar.js [hunk 1]
```diff
@@ -149,6 +149,8 @@ export class NivoResponsiveBar extends React.Component {
       </div>
     );
 
+    console.log(tooltip);
```
Comment (ktw1016) [inline client/src/charts/wrapped_nivo/nivo_bar.js:152]: Missed removing this one!

Comment (AmndaPlant) [reply client/src/charts/wrapped_nivo/nivo_bar.js:152]: Oops!

**Comment by Stephen-ONeil** [comment]
> All looks good, just wondering about `historical_auth_exp/auth_exp_prog_spending.js`
> Is this outdated version of `auth_exp_planned_spending`? @Stephen-ONeil

Oh, odd! Totally looks like it. The YAML file it tries to import doesn't exist, and it's not imported anywhere itself. Gonna say it's likely this was renamed/rewritten in to `auth_exp_planned_spending,js`, and that the original `auth_exp_prog_spending.js` was probably revived by a rebase conflict. I say delete it!

Also, oops, haha, this bug was totally on me, not updating all of these class names properly was sloppy. Thanks for doing the digging to find the commit that caused this and everything, Amanda!

**Comment by Stephen-ONeil** [comment]
Aha, looks like a false negative on the CI reporting as failed. Not the fault of this branch, just that the dev mongo is at capacity again.

**Comment by Stephen-ONeil** [comment]
Spoke too soon! This second failure may or may not be legitimate.

**Comment by AmndaPlant** [comment]
Yeah after a rerun it passed.

**Comment by Stephen-ONeil** [comment]
Good to rebase and merge then!

**Pull request merged**